### PR TITLE
logging enhancement

### PR DESCRIPTION
**Pull request by YuTaeseong**
#2 logging enhancement.

I use pino.

frontend
 pino

server
 pino
 pino-elasticsearch
 pino-multi-stream
 @elastic/ecs-pino-format

frontend

In Chrome Browser
image

server

In terminal
image

In Kibana
image

Let me know if you want more enhancement

### CODE REVIEW

File: server/src/config/index.ts [hunk 1]
```diff
@@ -5,6 +5,8 @@ const isProduction = process.env.NETTU_IS_PRODUCTION === 'true';
 const baseFrontendURL = process.env['FRONTEND_URL']!;
 const baseServerURL = process.env['SERVER_URL']!;
 
+const elasticsearchURL = process.env['NETTU_Elastic_URL'];
```
Comment (fmeringdal) [inline server/src/config/index.ts:8]: Please change `NETTU_Elastic_URL` to `NETTU_ELASTIC_URL`

File: server/src/logger.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import pino from 'pino';
+import { elasticsearchURL, isProduction } from './config';
+
+const ecsFormat = require('@elastic/ecs-pino-format');
+const pinoElastic = require('pino-elasticsearch');
+const pinoms = require('pino-multi-stream');
+
+const pinoMultiStream = pinoms.multistream;
+
+const streamToElastic = pinoElastic({
```
Comment (fmeringdal) [inline server/src/logger.ts:10]: Only add `streamToElastic` if `elasticsearchURL` is not null. If `elasticsearchURL` is null then stream to stdout even if in production

**Review by fmeringdal — CHANGES_REQUESTED** [review]
Hi @YuTaeseong ,

Thanks for your PR. It looks good, but just needs a couple of changes before it can be approved :)

**Comment by YuTaeseong** [comment]
> Hi @YuTaeseong ,
> 
> Thanks for your PR. It looks good, but just needs a couple of changes before it can be approved :)

I've done your request!!

**Review by fmeringdal — APPROVED** [review]
Looks good!

**Pull request merged**